.buildkite/pipelines/quality-gates/pipeline.tests-production.yaml

```
@@ -1,8 +1,18 @@
-# These pipeline steps constitute the quality gate for your service within the production
-# environment. Incorporate any necessary additional logic to validate the service's integrity.
+# These pipeline steps constitute the quality gate for your service within the production environment.
+# Incorporate any necessary additional logic to validate the service's integrity.
 # A failure in this pipeline build will prevent further progression to the subsequent stage.
 
 steps:
+  - label: ":pipeline::kibana::seedling: Trigger SLO check"
+    trigger: "serverless-quality-gates" # https://buildkite.com/elastic/serverless-quality-gates
+    build:
+      message: "${BUILDKITE_MESSAGE} (triggered by pipeline.tests-production.yaml)"
+      env:
+        TARGET_ENV: ${ENVIRONMENT}
```

Then this should also probably be hard-coded
```suggestion
        TARGET_ENV: production
```